js/ui/lookup.js

```diff
@@ -1129,6 +1090,12 @@ const Lookup = DropDownList.inherit({
             case 'cleanSearchOnOpening':
             case '_scrollToSelectedItemEnabled':
                 break;
+            case 'dropDownOptions.width':
```

Comment:
why? When user set option to `auto` he expect option to get 'auto' value, not lookup element width

Reply:
https://github.com/DevExpress/DevExtreme/blob/21_2/testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js#L1461

Reply:
oh.. ok, let's leave it as it is..